rfcs/2021-01-05-typography-scaling.md

```diff
@@ -0,0 +1,200 @@
+
+# Basic Example
+This is how the typography will look after the implementation of type scales.
+
+#### **On Desktop**
+<img width="600" src="./images/typography/website-example-desktop.png" />
+<img width="600" src="./images/typography/dashboard-example-desktop.png" />
+
+---
+
+#### **On Mobile**
+<div style="display:flex; align-items: flex-start">
+<img width="300" src="./images/typography/website-example-mobile.png" />
+<img width="300" src="./images/typography/dashboard-example-mobile.png" />
+</div>
+
+<br>
+
+# Motivation
+### **Why do we need a type scale?**
+We need our users to access our products anywhere, anytime. To make sure they can do so, we need to be present on almost every device possible especially on portables (mobiles & tablets). Every device offers a different experience and we need to make sure that we are flexible in that case. One of the major aspects of that experience is **readability** and therefore we are focusing on **typography scaling** across multiple devices.
+
+### **Use Cases**
+There can be several use-cases when a user wants to access the dashboard on multiple devices. Few of those can be,
+- **On Mobiles** (user can check for settlement numbers while commuting/traveling in a metro/bus)
+- **On Tablets** (user can capture a payment while on-the-go to their home back from the office)
+- **On Desktops** (user can request for access to the international payment via the dashboard)
+
+Or in-general a user can visit the company's marketing/landing page and consume information about us and our offerings.
+
+### **Examples from the current system**
+Here are a few examples in the current system.
+- Dashboard Route - Desktop vs. Mobile:
+![Dashboard Route: Desktop vs. Mobile](./images/typography/dashboard-route-mobile-desktop-comparison.png)
+
+- Dashboard Invoices - Desktop vs. Mobile:
+![Dashboard Invoices: Desktop vs. Mobile](./images/typography/dashboard-invoices-mobile-desktop-comparison.png)
+
+- Dashboard Invoices (Mobile) - Font Ratios:
+![Dashboard Invoices (Mobile): Font Ratios](./images/typography/dashboard-invoices-mobile-problems.png)
+
+### **Expected Outcome**
+A typography system that is more readable/consumable across different devices (mobiles/tablets/desktops).
+
+# Detailed Design
+
+### **Introduction**
+We are following a [**Major Second**](https://type-scale.com/?size=14&scale=1.125&text=A%20Visual%20Type%20Scale&font=Lato&fontweight=400&bodyfont=Poppins&bodyfontweight=400&lineheight=1.65&backgroundcolor=white&fontcolor=%23333&preview=false) modular scale (`1.125`) for our typography system. Simply put, the font-sizes will be derived in multiples of 1.125. Also, we will be using [**Lato**](https://fonts.google.com/specimen/Lato?sidebar.open=true&selection.family=Lato:wght@400;700) as our primary font with **Regular(400)** & **Bold(700)** weights *(as of now)*.
```

Comment:
We shall also add the details like what is the fallback font we are using. This is required when we are downloading a font at that time we need to show a system fallback font. 

Read more about [fallback fonts](https://css-tricks.com/css-basics-fallback-font-stacks-robust-web-typography/) and we can use [this tool](https://meowni.ca/font-style-matcher/) to generate a fallback font for us